src/cc65/coptind.c

```diff
@@ -1069,8 +1069,8 @@ unsigned OptTransfers4 (CodeSeg* S)
 
 
 
-unsigned OptPushPop (CodeSeg* S)
-/* Remove a PHA/PLA sequence were A is not used later */
+unsigned OptPushPop1 (CodeSeg* S)
```

Comment:
Maybe `OptPushPopAccumulator` and `OptPushPopStatus` would be more indicative names, but I see there's precedent for the numbered approach.

Reply:
Since the two have clear comments and are only used once each in one place, I don't think the current naming convention is unacceptable.